pennylane_sf/gbs.py

```diff
@@ -0,0 +1,258 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Strawberry Fields variational GBS device
+========================================
+
+**Module name:** :mod:`pennylane_sf.gbs`
+
+.. currentmodule:: pennylane_sf.gbs
+
+The Strawberry Fields variational GBS device is a simulator that provides a way to encode
+variational parameters into GBS so that the gradient with respect to the output probability
+distribution is accessible.
+
+Classes
+-------
+
+.. autosummary::
+   StrawberryFieldsGBS
+
+Code details
+~~~~~~~~~~~~
+"""
```

Comment:
Worth double checking, but I don't think this is displayed in Sphinx anymore. I would remove all the Sphinx-specific markup, and simply have
```python
"""
This module contains the variational GBS device.

The Strawberry Fields variational GBS device is a simulator that provides a way to encode
variational parameters into GBS so that the gradient with respect to the output probability
distribution is accessible.
"""